changelog/issue-1754.md

```diff
@@ -0,0 +1,4 @@
+level: minor
```

Comment:
`Silent` would suit better here

Reply:
@vishakhanihore Oh, ok, thank you! Will change it in next commit

Reply:
@vishakhanihore Done!